pathways_vue/components/search/search.vue

```diff
@@ -66,22 +59,35 @@ export default {
       }
       this.syncSelected();
     },
-    labelToKey(label) {
-      let keyArr = Object.entries(this.options)
-        .find(([_, value]) => value.label == label);
-      return keyArr[0];
-    },
+    labelFromKey(key) {
+      let label = "";
+      this.options.forEach(function (option, idx){
+          if(option['key'] === key){
+            label = option['value'];
+          }
+        });
+      return label;
+    }
   },
   computed: {
     renderableOptions() {
-      return Object.entries(this.options).map(([_, value]) => value.label);
+      return this.options.map(m => m.value)
```

Comment:
Course page seems to break on this line.

Reply:
Yeah, courses are going to need a refactor too.  I'm trying to get the major work merged due to the amount of changes included, it's becoming difficult to keep up with develop.  Any issues with course will be resolved as I work through refactoring course and adding in the data.